contents/docs/libraries/shopify.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: Shopify
+icon: ../../images/docs/integrate/frameworks/shopify.svg
+---
+
+import eventLight from '../../images/tutorials/shopify/event-light.png';
+import eventDark from '../../images/tutorials/shopify/event-dark.png';
+
+Tracking your [Shopify](https://www.shopify.com/) store with PostHog can help you improve the user experience and increase conversion rates.
+
+## Setting up a custom web pixel
+
+The simplest way to add PostHog to Shopify is using a [custom web pixel](https://shopify.dev/docs/api/web-pixels-api#custom-web-pixels). To set this up, first, go to the [customer events](https://admin.shopify.com/settings/customer_events) tab in your store settings, click "Add custom pixel," and give your pixel a name.
+
+Next, add code that contains:
+
+1. The customer events you want to subscribe to. You can find a complete list in Shopify's [Web Pixels API documentation](https://shopify.dev/docs/api/web-pixels-api/standard-events).
+
+2. Your PostHog JavaScript snippet which you can get from [your project settings](https://us.posthog.com/settings/project#snippet).
+
+3. The rest of your PostHog capture logic. 
+
+For example, we use a custom pixel in [our merch store](https://merch.posthog.com/) to identify customers and capture a `merch store order submitted` event with details like `userID`, `order_number`, `product_id`, `price`, and more.
```

Comment:
```suggestion
As an example, here is the code for subscribing to a `checkout_completed` event. Every time a user checks out, we capture an event. We also call [`posthog.identify()`](/docs/product-analytics/identify) so that we can track the user across different sessions:
```